view/frontend/web/js/view/payment/method-renderer/cc-method.js

```diff
@@ -304,7 +304,22 @@ define(
              * @override
              */
             getData: function () {
-                
+                if (this.intigrationmode === 'direct' && this.isClientSideEncryptionEnabled()) {
+                    return {
+                        'method': "worldpay_cc",
+                        'additional_data': {
+                            'cc_type': this.getselectedCCType('payment[cc_type]'),
+                            'cc_exp_year': this.creditCardExpYear(),
```

Comment:
Shouldn't we be removing the expiry? @AStevensTaylor

Reply:
Good spot, that shouldn't be there. Removed